tests/Language/VisitorTest.php

```diff
@@ -68,7 +68,8 @@ private function checkVisitorFnArgs(DocumentNode $ast, array $args, bool $isEdit
         if ($parent instanceof NodeList) {
             self::assertEquals($node, $parent[$key]);
         } else {
-            self::assertEquals($node, $parent->{$key});
+            $vars = get_object_vars($parent);
```

Comment:
This isn't any more type-safe than what was here before and is probably less performant. Any better ideas?

Reply:
Perhaps check `propery_exists()` before to ensure the dynamic access is fine.

Reply:
Hmm, wouldn't that make it harder to catch runtime errors?

Reply:
Well check and fail the test if it does not exist.

Reply:
We can also just ignore phpstan here.

Reply:
SGTM. bd02774f